capitalize_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './base_decorator'
+
+class BaseDecorator < BaseDecorator
```

Comment:
- [ ] Inheritance is not properly established. `CapitalizeDecorator` inheriting from `BaseDecorator` should be similar to:
```ruby
class CapitalizeDecorator < BaseDecorator
```

Kindly ensure to improve upon the same.